.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: FreeBSD Test
+
+on: push
+
+jobs:
+  test-freebsd:
+    runs-on: macos-latest
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: test
+      uses: vmactions/freebsd-vm@v0.1.2
+      with:
+        usesh: true
+        sync: rsync
+        prepare: pkg install -y bash gmake lang/tcl86
+        run: >
+          gmake &&
+          ./runtest --accurate --verbose --no-latency &&
+          MAKE=gmake ./runtest-moduleapi --verbose &&
+          ./runtest-sentinel &&
+          ./runtest-cluster
```

Comment:
why not use a separate `step` for each like the other jobs do?
is it because you'll have to repeat the `usesh` and `sync` in each?

Reply:
You can't, this is not a GitHub action step but configuration passed to the vmactions action which simply sshs into the VM and executes it. If you include several vmaction steps the VM will be teared down and set up again.